source/plugins/json/index.md

```diff
@@ -380,6 +380,49 @@ The implementation should then be defined in `struts.xml` like:
 </struts>
 ```
 
+There is an example at [struts-examples/json-customize/FlexJSONWriter.java](https://gitbox.apache.org/repos/asf?p=struts-examples.git;a=blob_plain;f=json-customize/src/main/java/org/demo/FlexJSONWriter.java;hb=HEAD).
+It replaces Strut's default json serializer with [Flexjson](http://flexjson.sourceforge.net/) as below:
```

Comment:
`Struts'`?

Reply:
👍 my childhood English teacher was wrong 😉 what do you think to delete `'` at all? i.e. `replaces Struts default json serializer`.

Reply:
👍